pkg/volume/scaleio/sio_volume_test.go

```diff
@@ -384,7 +384,7 @@ func TestVolumeProvisioner(t *testing.T) {
 	}
 	sioVol.sioMgr.client = sio
 	if err := deleter.Delete(); err != nil {
-		t.Fatalf("failed while deleteing vol: %v", err)
+		t.Fatalf("failed while deleting vol: %v", err)
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-41c44732f8236b99697b823607366ed7.